src/ucp/core/ucp_ep.c

```diff
@@ -526,21 +524,20 @@ ucs_status_t ucp_ep_create_server_accept(ucp_worker_h worker,
     }
 
     /* common non-CM flow */
-    ucs_assert(!ucp_worker_sockaddr_is_cm_proto(worker));
-out_free_address:
+    status = uct_iface_accept(conn_request->uct.iface,
```

Comment:
maybe move the assertion ucs_assert(!ucp_worker_sockaddr_is_cm_proto(worker)); to earlier place in the exit flow

Reply:
but then it's useless in case of error.

Reply:
move it to be first thing in error flow, instead of last thing

Reply:
what about error-free flow? assume it alway-true-error-free? :)

Reply:
ok